api/v1beta1/aerospikecluster_validating_webhook.go

```diff
@@ -449,6 +453,20 @@ func (c *AerospikeCluster) validateRackConfig(aslog logr.Logger) error {
 
 		config := rack.AerospikeConfig
 
+		// Validate rack aerospike config
+		if len(rack.AerospikeConfig.Value) != 0 {
+			if err := validateAerospikeConfigSchema(
+				aslog, version, config,
+			); err != nil {
+				// Differentiating rack aware config log message.
+				if rack.ID > 0 {
```

Comment:
Use this single error.
aerospikeConfig is not valid for the rack. Please check global and local aerospikeConfig for rack